header-rewrite-filter/utility.cc

```diff
@@ -46,6 +46,27 @@ BooleanOperatorType StringToBooleanOperatorType(absl::string_view bool_operator)
     }
 }
 
+FunctionType StringToFunctionType(absl::string_view function) {
+    if (function.compare(DYNAMIC_VALUE_HDR) == 0) {
+        return FunctionType::GetHdr;
+    } else if (function.compare(DYNAMIC_VALUE_URL_PARAM) == 0) {
+        return FunctionType::Urlp;
+    } else {
+        return FunctionType::InvalidFunctionType;
+    }
+}
+
+absl::Status GetFunctionArgument(FunctionType function_type, std::string& function_expression){
```

Comment:
Should we be using FunctionType here?

Reply:
I don't think it's necessary to use FunctionType, each dynamic function can use the argument in the way it needs.